rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -1091,6 +1095,12 @@ def main():
         type=str,
         help='Specify a desired branch of the corresponding git',
         default='master')
+    parser.add_argument(
```

Comment:
This is conered by #159

Reply:
So, per the description [here](https://github.com/cloud-bulldozer/perfscale-managed-services/pull/162#issue-1767697032): we need #159 first, I did this so I can avoid rebase related changes :P

Reply:
we need that the ticket will be verified during the argument verification. If we wait until _build_cluster, we will have a lot of stuff already created.

please add a type=check_function and check that the ticket is valid during the argparse phase

Reply:
Ack, will update the code

Reply:
Updated, ptal